portal-web/test/functional/com/liferay/portalweb/macros/KBTemplate.macro

```diff
@@ -25,6 +25,25 @@ definition {
 			rowEntry = "${kbTemplateTitle}");
 	}
 
+	macro gotoTemplateMenuItemThroughPageTreePanel {
+		MouseOver(
+			key_itemLinkName = "${kbTemplateTitle}",
+			locator1 = "KnowledgeBase#PAGE_TREE_ITEM_LINK");
+
+		Click.waitForMenuToggleJSClick(
+			key_itemName = "${kbTemplateTitle}",
+			locator1 = "KnowledgeBase#PAGE_TREE_ITEM_ELLIPSIS");
+
+		WaitForLiferayEvent.initializeLiferayEventLog();
+
+		if ("${menuItem}" == "Delete") {
+			MenuItem.clickNoError(menuItem = "${menuItem}");
+		}
+		else {
+			MenuItem.click(menuItem = "${menuItem}");
```

Comment:
Is this conditional necessary?

Reply:
Yeah, We need to click below confirmation info when we delete an entry. So we can only use `MenuItem.clickNoError`.
![image](https://user-images.githubusercontent.com/48118285/193188338-5a456a64-40e8-4917-a2df-6374d14f3640.png)